File: an2_sem4/MPP/Homework/sapt9/cs/FestivalSellpoint/FestivalSellpoint.Network/ObjectProtocol/ErrorResponse.cs
using System;

namespace FestivalSellpoint.Network.ObjectProtocol
{
    [Serializable]
    public class ErrorResponse : IResponse
    {
        public string Message { get; set; }
        public ErrorResponse(string message) => Message = message;        
        public override string ToString() => $"ErrorResponse {{ Message = {Message} }}";
    }
}

File: an2_sem4/MPP/Homework/sapt11/java/network/src/main/java/ro/ubbcluj/cs/stefnmubb/festivalsellpoint/network/objectprotocol/ErrorResponse.java
package ro.ubbcluj.cs.stefnmubb.festivalsellpoint.network.objectprotocol;

public class ErrorResponse implements IResponse {
    private String message;

    public ErrorResponse() { this.message=""; }

    public ErrorResponse(String message) {
        this.message = message;
    }
    public ErrorResponse(Exception e) {
        this.message = e.toString();
    }

    public String getMessage() {
        return message;
    }

    @Override
    public String toString() {
        return "ErrorResponse{" +
                "message='" + message + '\'' +
                '}';
    }
}

